pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -128,7 +128,7 @@ public void deleteNamespace(@PathParam("property") String property, @PathParam("
     }
 
     @DELETE
-    @Path("/{property}/{namespace}/bundle/{bundle}")
```

Comment:
hmm this sounds like a breaking change, no?

Reply:
This is only relative to v2 namespace. It was broken when v2 handler created.